app/src/main/java/com/aragh/kotlin2/screen/albumdetails/AlbumDetailsContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.aragh.kotlin2.screen.albumdetails
+
+
```

Comment:
I've encountered a dozen of times a different approach
```
interface AlbumDetailsContract {
  interface View {
    ...
  }

  interface Presenter {
    ...
  }
}
```

by doing this, you wouldn't have conflicting imports for `PresenterContract` in the DI.